conduce/api.py

```diff
@@ -756,6 +778,128 @@ def delete_transactions(dataset_id, **kwargs):
     return make_delete_request(fragment, **kwargs)
 
 
+def add_simple_store(dataset_id, auto_process, **kwargs):
+    """
+    Adds a simple store to the specified dataset.
+
+    If configured for auto-processing (default), transactions will not be auto-processed
+    into the store until new transactions are posted.
+
+    Parameters
+    ----------
+    dataset_id : string
+        The UUID that identifies the dataset to modify.
+    auto_process : boolean
+        Configure the backend to auto process transactions. Auto processing is enabled by default.
+        Set to False to disable auto processing.
+    **kwargs : key-value
+        See :py:func:`make_post_request` for more kwargs.
+    """
+    return _create_dataset_backend(dataset_id, DatasetBackends.simple, auto_process, None, **kwargs)
+
+
+def add_tile_store(dataset_id, auto_process, **kwargs):
+    """
+    Adds a tile store to the specified dataset.
+
+    If configured for auto-processing (default), transactions will not be auto-processed
+    into the store until new transactions are posted.
+
+    Parameters
+    ----------
+    dataset_id : string
+        The UUID that identifies the dataset to modify.
+    auto_process : boolean
+        Configure the backend to auto process transactions. Auto processing is enabled by default.
+        Set to False to disable auto processing.
+    **kwargs : key-value
+        See :py:func:`make_post_request` for more kwargs.
+    """
+    return _create_dataset_backend(dataset_id, DatasetBackends.tile, auto_process, None, **kwargs)
+
+
+def add_capped_tile_store(dataset_id, auto_process, minimum_temoral_level, maximum_temporal_level, **kwargs):
```

Comment:
`minimum_temoral` level should be `minimum_temporal_level`

Reply:
`minimum_temporal_level` and `maximum_temporal_level` are not documented in the parameters section.

Reply:
Wondering if `maximum_temporal_zoom` would be more clear.  Maybe `max_temporal_zoom_level`.

Reply:
Since I botched those parameters, I did wind up changing them to shorten to min/max and include the word zoom for clarity.